develop/blockchains/get-started/build-custom-blockchains.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Build Custom Blockchains
+description: Learn how to build custom blockchains using the Polkadot SDK, focusing on pre-built chain templates for faster development.
+---
+
+# Build Custom Blockchains
+
+## Overview
+
+Building custom blockchains with the [Polkadot SDK](https://github.com/paritytech/polkadot-sdk/tree/master){target=\_blank} allows developers to create specialized blockchain solutions tailored to unique requirements. By leveraging [Substrate](https://github.com/paritytech/polkadot-sdk/tree/master/substrate){target=\_blank}—a Rust-based, modular blockchain development framework—the Polkadot SDK provides powerful tools to construct chains that can either stand-alone or connect to Polkadot’s shared security network as parachains. This flexibility empowers projects across various sectors to launch blockchains that meet specific functional, security, and scalability needs.
+
+This guide covers the core steps for building a custom blockchain using the Polkadot SDK, starting from pre-built chain templates. These templates simplify development, providing an efficient starting point that can be further customized, allowing you to focus on implementing the features and modules that set your blockchain apart.
+
+## Starting from Templates
+
+Using pre-built templates is an efficient way to begin building a custom blockchain. Templates provide a foundational setup with pre-configured modules, letting developers avoid starting from scratch and instead focus on customization. Depending on your project’s goals—whether you want a simple test chain, a standalone chain, or a parachain that integrates with Polkadot’s relay chains—there are templates designed to suit different levels of complexity and scalability.
+
+Within the Polkadot SDK, the following templates are available to get you started:
+
+- [minimal-template](https://github.com/paritytech/polkadot-sdk/tree/master/templates/minimal){target=\_blank} - includes only the essential components necessary for a functioning blockchain. It’s ideal for developers who want to gain familiarity with blockchain basics and test simple customizations before scaling up
```

Comment:
Maybe format: `minimal-template`

Reply:
yeah, it's better that way! fixed! thanksss